bin/maildev

```diff
@@ -3,6 +3,9 @@
 var path = require('path');
 var fs = require('fs');
 
+process.on('SIGINT', process.exit); // Handle ctrl-c when running via docker run
+process.on('SIGTERM', process.exit); // Handle when we docker rm the container
```

Comment:
Please don't do that here. Or maybe, do that here, but please have a look at the index.js. There is already a listener for SIGTERM.